sensitive-terms.txt

```diff
@@ -0,0 +1,1933 @@
+@55
+@ssfcker
+@ssfucker
+@ssfvcker
+@sshole
+🖕
+0ral seks
+0ral sex
+0rg@sm
+0rgasms
+2 girls 1 cup
+2g1c
+3jakulating
+4r5e
+4r5ed
+4r5es
+4skin
+5h17
+5h1t
+5kank
+69
+a_s_s
+abbie
+abeed
+abo
+aboe
+acrotomophilia
+alabama hot pocket
+alaskan pipeline
+anal
+anal hole
+anal hore
+anal pirate
+analingus
+analplug
+analplugs
+anilingus
+animal fucker
+anus
+anus plug
+ape shit
+ape shite
+apeshit
+apeshite
+ar5e
+ar5ehole
+arse
+arse gobbler
+arse hole
+arse-bandit
+arse-bandits
+arse-fucker
+arse-fuckers
+arse-hole
+arse-shagger
+arsefuck
+arsefucker
+arsehole
+arseshit
+arsewipe
+ashole
+ass
+ass bandit
+ass eating boiolas
+ass eating nob jokeys
+ass fecker
+ass fuck
+ass fucker
+ass fucking boiolas
+ass fucking nob jokeys
+ass hole
+ass kisser
+ass licking
+ass muncher
+ass-bandit
+ass-fucker
+ass-fuckers
+ass-hole
+asses
+assface
+assfck
+assfcker
+assfk
+assfkcer
+assfker
+assfkr
+assfook
+assfuccer
+assfuck
+assfuck3r
+assfucker
+assfuckers
+assfuckerz
+assfukah
+assfukka
+assfukker
+assfukkerz
+assfvck3r
+assfvcker
+asshat
+asshol3
+asshole
+asshole fucker
+assholes
+assmunch
+assmuncher
+asswhole
+asswipe
+auto erotic
+autoerotic
+axe wound
+azz
+azzhole
+b to the inch
+b！tch
```

Comment:
Why don't we have `b!tch`, too, without a space?

Reply:
The space might be a typo, I'll fix. thanks!